### ISSUE TITLE
Empty editor when using hass-cli state edit <entity_id>

### ISSUE THREAD

**Issue by VDRainer** [home-assistant/home-assistant-cli#201]
```
(hass-cli) rk:~$ hass-cli --version
hass-cli, version 0.6.0
(hass-cli) rk:~$ hass-cli info
BASE_URL                   LOCATION    REQUIRES_API_PASWORD    VERSION
http://192.168.77.67:8123  Home        False                   0.89.0b2
```
Tried with nano and vi.

**Comment by maxandersen** [comment]
Can you run edit with hass-cli --debug and show the output? (Might need to scrub tokens/ips)

**Comment by VDRainer** [comment]
Thanks @maxandersen for looking at this.
Here's the output:
```
(hass-cli) rk:~$ hass-cli --debug state edit light.milight_gu10 
debug: Setting auto-output to: yaml
debug: Session: verify(True), cert(None)
debug: Starting new HTTP connection (1): 127.0.0.1:8123
send: b'GET /api/states/light.milight_gu10 HTTP/1.1\r\nHost: 127.0.0.1:8123\r\nUser-Agent: python-requests/2.21.0\r\nAccept-Encoding: gzip, deflate\r\nAccept: */*\r\nConnection: keep-alive\r\nContent-Type: application/json\r\nAuthorization: Bearer xxxx\r\n\r\n'
reply: 'HTTP/1.1 200 OK\r\n'
```
- nano starts empty `/tmp/editor-7pae0oc1.yaml` (should this be json?)
- closing nano without save
```
debug: http://127.0.0.1:8123 "GET /api/states/light.milight_gu10 HTTP/1.1" 200 211
header: Content-Type header: Content-Length header: Content-Encoding header: Date header: Server No edits/changes returned from editor.
```

**Comment by maxandersen** [comment]
reproduced, fixed via #202
